Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY . .
 RUN make
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
-ARG CSV=4.1
+ARG CSV=4.2
```

Comment:
@ewolinetz @jcantrill 
it's a good thing I deleted the 4.1 dir or this might have been overlooked.......